tests/PySys/mqtt_port/mqtt_port_change_connection_works/run.py

```diff
@@ -0,0 +1,115 @@
+import sys
+import time
+
+from pysys.basetest import BaseTest
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system, that is configured with certificate created and registered in a cloud
+When the thin edge device is disconnected from cloud, `sudo tedge disconnect c8y`
+When `tedge mqtt.port set` with sudo
+When the thin edge device is connected to cloud, `sudo tedge connect c8y`
+Now validate the services that use the mqtt port
+   Validate tedge mqtt pub/sub
+   Validate tedge connect c8y --test
+   Validate tedge_mapper status
+
```

Comment:
Why `collectd` and `collectd_mapper` were excluded?

Reply:
The collectd and the collect_mapper are not part of the test because updating the port in collectd is a manual step. I will update the test to add collectd_mapper. Just to check the connection is working or not.